src/python/zigopt/api/request.py

```diff
@@ -253,8 +254,8 @@ def parse_json(self):
 
   def optional_param(self, name):
     """Retrieve ``name`` from flask HTTP request."""
-    unvalidated = self.params().get(name)
-    return napply(unvalidated, lambda v: validate_type(v, ValidationType.string, key=name))
+    unvalidated: Any | None = self.params().get(name)
```

Comment:
`Any` doesn't  include `None`? I would've thought marking something with `Any` meant it didn't really need any other typing info?

Reply:
This seems to suppress the error (something about not being able to resolve the type). I chose `| None` because the `napply` below applies the function only if the argument is not `None`.